README.md

```diff
@@ -158,6 +146,15 @@ You will need to provide the IP that the GUI will work with. By running this scr
 ./robot/basestation/env.sh >| robot/basestation/static/js/env.js
 ```
 
+### Setup git hooks
+
+Git hooks are important for performing repository validity checks. To setup git hooks using Git Bash, run the following commands from the root of the repository (ex: from ~/Programming/robotics-prototype/):
+
+- `cp commit_message_hook.py .git/hooks/prepare-commit-msg`
+- `cp branch_name_verification_hook.py .git/hooks/post-checkout`
+
+If you're on windows, install [Git Bash](https://git-scm.com/downloads) to be able to run the same commands.
```

Comment:
windows?? If they are inside of docker they will still use the hooks inside the cloned repo which is linux

Reply:
Idk man it was already there

Reply:
removed that line cause we shouldnt be using windows lel
docker pr can add instructions for different platforms